### ISSUE TITLE
auto logout when browser is closed

### ISSUE THREAD

**Issue by jungle-boogie** [bitwarden/core#147]
Hi All,

I would like to be logged out of my bitwarden account when my browser is closed after a small duration of time, probably 90 seconds to 2 minutes. It's very simple to sign back into BW, so I'd rather do that than have BW keep me logged in, even if I need to type in the password.

Similar, but separate issue: https://github.com/bitwarden/core/issues/146

Thanks for the consideration!

**Comment by kspearrin** [comment]
Closing the browser tab or browser window will already log you out...?

**Comment by jungle-boogie** [comment]
@kspearrin, are you saying if the browser is closed for 10 seconds, I'd get logged out of BW?

**Comment by kspearrin** [comment]
Correct. You don’t even have to close the browser. Just close the tab. It’s session storage only.

**Comment by jungle-boogie** [comment]
Are you saying closing a tab in a browser will auto log me out of bitwarden?

**Comment by kspearrin** [comment]
Yes

**Comment by jungle-boogie** [comment]
I haven't seen that happen. How do I verify that?

**Comment by jungle-boogie** [comment]
I've signed into BW, signed into reddit via BW, closed the tab, went a few sites that I don't have in BW, and noticed I was NOT logged out of BW.

**Comment by jungle-boogie** [comment]
Firefox 57 on openBSD -current

**Comment by kspearrin** [comment]
Are we taking about using the web vault here or the browser extension?

**Comment by jungle-boogie** [comment]
Ah, that's possibly the issue. I'm talking about the browser extension. Should I have opened this under that project?

**Comment by kspearrin** [comment]
Yes this project is for the apis.

**Comment by jungle-boogie** [comment]
Sorry! :( closing!
